map_model/src/edits/mod.rs

```diff
@@ -606,6 +621,88 @@ fn recalculate_intersection_polygon(
     affected
 }
 
+fn fix_buildings(map: &mut Map, input: Vec<BuildingID>) {
```

Comment:
Maybe my blood sugar is low, but I'm having a hard time understanding what all this function does.

After staring at the code a bit - I think specifically what we're "fixing" is that the map isn't fully initialized yet - we need to calculate a connection from a building to the transportation network by trying to produce both a valid `sidewalk_pos` and `driveway_geom` for each building, is that right? 

Anything else?

Reply:
The map is already fully initialized, but we just broke it by editing. Renaming this `fix_building_driveways` and adding a doc comment